lib/internal/fs/promises.js

```diff
@@ -252,6 +258,40 @@ class FileHandle extends EventEmitterMixin(JSTransferable) {
     return readable;
   }
 
+  /**
+   * @typedef {import('./streams').ReadStream
+   * } ReadStream
+   * @param {{
+   *   encoding?: string;
+   *   autoClose?: boolean;
+   *   emitClose?: boolean;
```

Comment:
Can we remove autoClose & emitClose from here?

Reply:
They are legacy. The only reason we have still have them is to avoid breakage. Since this is a new API they are not necessary.

Reply:
It’s a new way for exposing the same old API, removing it would be harder than keeping it – also I don’t think they’re documented as legacy anywhere, so that’s probably best to leave that discussion for another PR.